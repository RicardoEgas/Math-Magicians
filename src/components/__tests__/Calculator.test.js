import React from 'react';
import renderer from 'react-test-renderer';
import Calc from '../Calculator';

describe('Home component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Calc />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
