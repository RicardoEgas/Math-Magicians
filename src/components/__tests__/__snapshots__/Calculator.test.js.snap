// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
Array [
  <h2
    className="math-header"
  >
    Lets do some math
  </h2>,
  <div
    className="calculator"
  >
    <div
      className="result"
    >
      0
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      AC
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      +/-
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      %
    </div>
    <div
      className="btn orange"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      รท
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      7
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      8
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      9
    </div>
    <div
      className="btn orange"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      x
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      4
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      5
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      6
    </div>
    <div
      className="btn orange"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      -
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      1
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      2
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      3
    </div>
    <div
      className="btn orange"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      +
    </div>
    <div
      className="btn zero"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      0
    </div>
    <div
      className="btn"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      .
    </div>
    <div
      className="btn orange"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="presentation"
    >
      =
    </div>
  </div>,
]
`;
